GOVERNANCE.md

```diff
@@ -1,18 +1,100 @@
 ---
 # SPDX-License-Identifier: CC0-1.0
-# SPDX-FileCopyrightText: 2019-2024 The Foundation for Public Code <info@publiccode.net>, https://standard.publiccode.net/AUTHORS
+# SPDX-FileCopyrightText: 2024 Standard for Public Code Authors, https://github.com/standard-for-public-code/standard-for-public-code/blob/develop/AUTHORS.md
 ---
-# Governance
 
-This standard lies at the core of the codebase stewardship provided by the [Foundation for Public Code](https://publiccode.net/).
-We decide if a codebase is ready for community co-development based on this document.
+# Governance.md
```

Comment:
This is a filename, I would keep the header 'Governance'.


```suggestion
# Governance
```

Reply:
I agree.